src/react/LoginPage.jsx

```diff
@@ -56,8 +56,8 @@ function tokenValidationFeedback(tokenValidation) {
 const TokenForm = ({ allowSwitching, defaultRealm, onSwitchLogin, onLogin }) => {
   const [realm, setRealm] = useState(defaultRealm);
   const [token, setToken] = useState('');
-  const isValidRealm = isValidRealmName(realm);
-  const tokenValidation = validateAstarteToken(token);
+  const isValidRealm = Astarte.utils.isValidRealmName(realm);
```

Comment:
we strongly advocate against any `utils` or too generic containers. if it contains realm infos, call it `Realm`.
If you still need generic helpers, try to be more specific. like `HtmlHelpers` or `JsonUtils` but let's avoid them unless necessary

Reply:
The `Realm` namespace seems nice but I was afraid it would be confused for a Class constructor, as we are already exporting classes such as `Astarte.Block`. Same goes for your comment on `Astarte.Token`. What's your take on it?